resources/test_data.json

```diff
@@ -467,5 +467,92 @@
         "note": "",
         "bug_type": "crash",
         "checker_qual_required": false
+    },
+    {
+        "issue_id" : "cf-3025",
+        "url": "https://github.com/tahiat/guava.git",
+        "issue_url": "https://github.com/typetools/checker-framework/issues/3025",
+        "branch": "cf-3025",
+        "commit_hash": "",
+        "project_name": "guava",
+        "build_command": "",
+        "root_dir": "guava/src/", 
+        "targets": [
+            {
+                "method": "difference(Map<? extends K, ? extends V>, Map<? extends K, ? extends V>, Equivalence<? super V>)",
+                "file": "Maps.java",
+                "package": "com.google.common.collect",
+                "inner_class": ""
+            }
+        ],
+        "cf_version": "3.0.1",
+        "java_version": "8",
+        "note": "",
+        "bug_type": "semi_crash",
+        "bug_pattern": {
+            "file_pattern": "Compilation unit: (.*)",
+            "error_pattern": "error: (.+?):",
+            "exception_pattern": "Exception: (.+):",
+            "stack_trace_pattern": "Stack trace: (.+)"
+        },
+        "checker_qual_required": true
+    },
+    {
+        "issue_id" : "jdk-8319461",
+        "url": "https://github.com/tahiat/property.git",
+        "issue_url": "https://bugs.openjdk.org/browse/JDK-8319461",
+        "test_case_url": "https://bugs.openjdk.org/browse/JDK-8319461",
+        "branch": "test",
+        "commit_hash": "",
+        "project_name": "property",
+        "build_command": "",
+        "root_dir": "core/src/main/java/", 
+        "targets": [
+            {
+                "method": "create(Class<P>, Class<V>, String ,PropertyMetadata<V>)",
```

Comment:
nit: the comma placement here seems wrong. It won't matter to Specimin (it removes spaces), but it would be easier to read this if we did it correctly.